runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -1693,6 +1693,19 @@ J9::ARM64::CodeGenerator::inlineDirectCall(TR::Node *node, TR::Register *&result
       {
       switch (methodSymbol->getRecognizedMethod())
          {
+         case TR::java_nio_Bits_keepAlive:
+         case TR::java_lang_ref_Reference_reachabilityFence:
+            {
+            TR::Node *paramNode = node->getFirstChild();
```

Comment:
If you copied this implementation from x86, can you copy the comments that were in that code as well?  It helps to explain what this code is trying to achieve.

Reply:
I find the placement of these comments to be a bit odd.  Can you move them down below the opening brace like x86 has them?  Also, remove the specific reference to `keepAlive` as this applies to a couple of different recognized methods.  Maybe say "these functions" instead?